### PR TITLE
Additional commands for prometheus and grafana

### PR DESCRIPTION
**Pull request by akremsa**
**What type of PR is this?**
> /kind cleanup

**What this PR does / Why we need it**:
While I was working on prometheus\grafana-related tasks I found that it would be nice to add some new commands inside a Makefile.

### CODE REVIEW

File: build/Makefile [hunk 1]
```diff
@@ -445,6 +445,14 @@ setup-prometheus:
 		$(HELM_ARGS) \
 		-f $(mount_path)/build/prometheus.yaml
 
+run-prometheus: $(ensure-build-image)
+	$(DOCKER_RUN) \
+		kubectl --namespace metrics port-forward $(shell kubectl get pods --namespace metrics -l 'app=prometheus,component=server' -o name) 9090
```
Comment (cyriltovena) [inline build/Makefile:450]: Don't we have a portforward target already for Prometheus and Grafana ?

https://github.com/googleforgames/agones/blob/b134abbd4f2e4a9ab9fd57d3cadf19821e4fb7e3/build/README.md#make-grafana-portforward

Comment (akremsa) [reply build/Makefile:450]: Ok, good. Didn't notice that.

**Comment by aLekSer** [comment]
Some E2E tests are failing due to change in consul lock name in https://github.com/googleforgames/agones/pull/1611 . Previously it was `LockE2E`, now we need to have separate two locks, one per each E2E test cluster name.

**Comment by aLekSer** [comment]
These two would be useful.

**Pull request merged**